lib/iris/coord_systems.py

```diff
@@ -1135,13 +1157,81 @@ def as_cartopy_crs(self):
             self.false_easting,
             self.false_northing,
             self.true_scale_lat,
+            self.scale_factor_at_projection_origin,
             globe=globe,
         )
 
     def as_cartopy_projection(self):
         return self.as_cartopy_crs()
 
 
+class PolarStereographic(Stereographic):
+    """
+    A subclass of the stereographic map projection centred on a pole.
+
+    """
+
+    grid_mapping_name = "polar_stereographic"
+
+    def __init__(
+        self,
+        central_lat,
+        central_lon,
+        false_easting=None,
+        false_northing=None,
+        true_scale_lat=None,
+        scale_factor_at_projection_origin=None,
+        ellipsoid=None,
+    ):
+        """
+        Construct a Polar Stereographic coord system.
+
+        Parameters
+        ----------
+
+        central_lat : {90, -90}
+            The latitude of the pole.
+
+        central_lon : float
+            The central longitude, which aligns with the y axis.
+
+        false_easting : float, optional
+            X offset from planar origin in metres.
+
+        false_northing : float, optional
+            Y offset from planar origin in metres.
+
+        true_scale_lat : float, optional
+            Latitude of true scale.
+
+        scale_factor_at_projection_origin : float, optional
+            Scale factor at the origin of the projection
+
+        ellipsoid : :class:`GeogCS`, optional
+            If given, defines the ellipsoid.
+
+        Notes
+        -----
+        It is only valid to provide one of `true_scale_lat` and `scale_factor_at_projection_origin`.
+        TODO: What if you give neither?
```

Comment:
Do you have an opinion on this @pp-mo? Can assume scale factor is 1, or refuse?

Reply:
Before we added the new "scale_factor" keyword, the parent class supported only `true_scale_lat`, which was strictly optional.  So that surely means that it should now still be correct to set _neither_, which would have the same meaning.
And it makes sense to me that PolarStereo should do just what Stereo does in this respect.
I think it is **_not_** correct to default a scale-factor to 1.0 --  the correct default is 'None' :  Then, what that _actually_ does, or what number it is equivalent to, is up to the lower-level calls (i.e. cartopy or proj).